frontend/src/pages/dashboard/dashboard.tsx

```diff
@@ -27,6 +31,20 @@ const dashboardItems = [
 
 
 export default function Dashboard() {
+
+    const { isLoggedIn, isLoading, handleSignOut } = useGlobalContext()
+    const navigate = useNavigate()
+
```

Comment:
Could switch this part to the `<AppCover />`  element to avoid glitches in the interface of dashboard.